src/cinder/qtime/QuickTimeImplAvf.mm

```diff
@@ -205,6 +205,10 @@ - (void)outputSequenceWasFlushed:(AVPlayerItemOutput *)output
 		[mAsset cancelLoading];
 		[mAsset release];
 	}
+
+	if( mPlayerVideoOutput ) {
```

Comment:
The conditional is unnecessary since messaging `nil` is fine, but in keeping with the proximate style.